### ISSUE TITLE
Enhancement: Vim Scrolling

### ISSUE THREAD

**Issue by xaviervalarino** [yoshuawuyts/vmd#91]
I love this tool and use it almost every day while looking over  READMEs in local node_modules. One thing that I personally would really enjoy—which is a bit of a pipe dream—would be  keyboard scrolling with `J` and `K` keys. (It would also be nice to be able to use `gg` and `G` as well, not to mention searching for text with `/`).

**Comment by maxkueng** [comment]
Great idea! There is already an issue for searching within a document #73. You can add a comment there about the `/` key and subscribe to notifications if you want to keep track.

**Comment by felixSchl** [comment]
@xaviervalarino Hey, this is not supposed to be a plug, but it might be of interest to you: https://github.com/felixSchl/gh-preview. I am not really maintaining that project anymore but I use it almost daily or whenever I have markdown work to do like writing my READMEs, changelogs and so on. You edit your markdown in vim and it renders it in the browser, looking as much as possible like if it was on github. Just thought I'd leave this here

**Comment by xaviervalarino** [comment]
@felixSchl I think you misunderstood. I would like to use Vim key bindings to scroll down the page, much like [Vimperator](http://www.vimperator.org/) or [Vimium](http://vimium.github.io/).

That being said, gh-preview reminds me of issue #82, where @davidmarkclements put forth the idea of having the the electron window scroll to where the cursor was in an open text editor. I can't tell from the gif if gh-preview supports scrolling on the browser window, but wonder if any of your work could be used to create that feature.

**Comment by sorrowchen** [comment]
@xaviervalarino
Yeah, feel the same way if this support vim-like movement, I am a new vmd user, currently I only found that 
SPC  next page 
Shift+SPC  previous page.
